cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,18 @@
 /// <reference types="cypress" />
-
 describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
-
-  });
+  beforeEach(() => {
+ cy.visit('https://react-redux.realworld.io')
+});
+it('should provide an ability to log in', () => {
+cy.contains('a', 'Sign in').click();
+cy.url().should('include','/login');
+cy.get(':nth-child(1) > .form-control')
+ .type('userNam@gmail.com');
+ cy.get(':nth-child(2) > .form-control')
+ .type('test1234');
+ cy.get('.btn').click();
+ cy.get('.navbar')
+ .should('contain.text', 'userna123' );
```

Comment:
Username to be expected after signing in is hard coded. This should ideally be a variable that corresponds to the user you are trying to log in.